tests/virt_autotest/guest_migration_src.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This test verifies guest migration between two different hosts, either xen to xen, or kvm to kvm.
+#          This is the part to run on the source host.
+# Maintainer: alice <xlai@suse.com>
+
+use base multi_machine_job_base;
+use strict;
+use testapi;
+use lockapi;
+use mmapi;
+
+sub get_script_run() {
+    my ($self) = @_;
+
+    my $dst_ip     = $self->get_var_from_parent('DST_IP');
+    my $dst_user   = $self->get_var_from_parent('DST_USER');
+    my $dst_pass   = $self->get_var_from_parent('DST_PASS');
+    my $guests     = get_var("GUEST_LIST", "");
+    my $hypervisor = get_var("HOST_HYPERVISOR", "kvm");
+    my $test_time  = get_var("MAX_MIGRATE_TIME", "10800") - 90;
+
+    my $pre_test_cmd
+      = "/usr/share/qa/tools/test_virtualization-guest-migrate-run -d $dst_ip -v $hypervisor -u $dst_user -p $dst_pass -i \"$guests\" -t $test_time";
```

Comment:
please extract variables and break lines to have shorter strings, e.g. all the arguments could go into an argument `$args`.